org.kde.kfloppy.json

```diff
@@ -0,0 +1,29 @@
+{
+    "id": "org.kde.kfloppy",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "5.15-21.08",
+    "sdk": "org.kde.Sdk",
+    "command": "kfloppy",
+    "rename-icon": "kfloppy",
+    "finish-args": [
+        "--share=ipc",
+        "--socket=fallback-x11",
+        "--socket=wayland",
+        "--filesystem=host"
```

Comment:
do you need full host filesystem access? It's my understand this is meant to format floppies, nothing else.